src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/TelemetryService.kt

```diff
@@ -1,7 +1,9 @@
-package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.config
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.service
```

Comment:
I moved the `TelemetryService` from the `config` package to the `service` package (because thats where it belongs 😁 )